### PR TITLE
Clarify parameter explanations of the Jira backend

### PR DESCRIPTION
**Pull request by isikozsoy**
This PR aims at providing a quick fix for the issue #752 by changing the explanation of the parameters for Jira and JiraClient backend.

### CODE REVIEW

File: perceval/backends/core/jira.py [hunk 1]
```diff
@@ -96,8 +96,8 @@ class Jira(Backend):
 
     :param url: JIRA's endpoint
     :param project: filter issues by project
-    :param user: Jira user
```
Comment (vchrombie) [inline perceval/backends/core/jira.py:99]: I would change this to
```
-      :param user: Jira user
+      :param user: Jira user (email)
```

File: perceval/backends/core/jira.py [hunk 2]
```diff
@@ -96,8 +96,8 @@ class Jira(Backend):
...
-    :param password: Jira user password
```
Comment (vchrombie) [inline perceval/backends/core/jira.py:100]: I would revert this change because the current implementation is based on password. It is better if we change it only after the implementation is changed to token based.

File: perceval/backends/core/jira.py [hunk 3]
```diff
@@ -273,8 +273,8 @@ class JiraClient(HttpClient):
 
     :param URL: URL of the JIRA server
     :param project: filter issues by project
-    :param user: JIRA's username
```
Comment (vchrombie) [inline perceval/backends/core/jira.py:276]: `username` might be misleading, so it would be great if we can follow the above comment
```
-      :param user: Jira user
+      :param user: Jira user (email)
```

File: perceval/backends/core/jira.py [hunk 4]
```diff
@@ -273,8 +273,8 @@ class JiraClient(HttpClient):
...
-    :param password: JIRA's password
```
Comment (vchrombie) [inline perceval/backends/core/jira.py:277]: same as the above comment, revert this change for now and change it only after implementation

**Review by vchrombie — CHANGES_REQUESTED** [review]
Thanks for opening the PR @isikozsoy.
I had some small comments on the PR, please let me know what do you think.

Also, it would be great if you can fix the sign-off issue and maybe amend the commits too. Please don't hesitate to ask for help.

File: perceval/backends/core/jira.py (hunk 2 above)
Comment (isikozsoy) [reply perceval/backends/core/jira.py:100]: When I used Jira backend in my project, I use email for the user field and api token for the password field. However, it did not work when I used email and password for authentication. Should I still revert this change?

**Comment by isikozsoy** [comment]
_Also, it would be great if you can fix the sign-off issue and maybe amend the commits too. Please don't hesitate to ask for help._

Could you guide me to fix the sign-off issue and to amend the commits please?

**Comment by isikozsoy** [comment]
_Also, it would be great if you can fix the sign-off issue and maybe amend the commits too. Please don't hesitate to ask for help._

Could you guide me to fix the sign-off issue and to amend the commits please?

File: perceval/backends/core/jira.py (hunk 2 above)
Comment (vchrombie) [reply perceval/backends/core/jira.py:100]: Since the current implementation is based on password, changing the comment to token might not look good. It would be really great if you can work on the feature of changing the basic_auth to token_auth. You can work in the same PR. WDYT?

**Comment by vchrombie** [comment]
Hi @isikozsoy 

> Could you guide me to fix the sign-off issue and to amend the commits please @vchrombie? I also commented on one of your change requests. Thanks.

Sure, you just need to amend the commit with `-s` flag and then force push the branch
```
$ git commit --amend -s
$ git push -f origin jira-backend-description
```

Hope this helps.

**Comment by sduenas** [comment]
This PR has been inactive for many years. Please feel free to reopen it if you resume your work on this.
